### ISSUE TITLE
Clr import system no module named System

### ISSUE THREAD

**Issue by satanasss** [pythonnet/pythonnet#388]
Hi guys, 

I'm trying to use to do an import System in my python code but it won't work. 

Here is what I do:
Import clr
Import System

>>>inportError: No module named System

So I tried a clr.AddReference, same thing, "AttributeError: 'module' object has no attribute 'AddReference' 

Have you guys already had this pb ? Do you need more info ?

Thanks in advance

### Environment

-   Pythonnet version: 2.2.2
-   Clr version 1.0.2
-   Python version: 2.7.1 anaconda 2.4.1
-   Operating System: windows

### Details

-   Describe what you were trying to get done.

    _TODO_

-   What commands did you run to trigger this issue? If you can provide a
    [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
    this will help us understand the issue.

```python
    print('TODO')
```

-   If there was a crash, please include the traceback here.

```python
    print('TODO')
```

**Comment by denfromufa** [comment]
Where did you get this `Clr version 1.0.2`?
Is it this package?

https://pypi.python.org/pypi/clr/1.0.3

If yes, you need to uninstall this clr package to use pythonnet, because
both packages have a name clash.

**Comment by denfromufa** [comment]
You can also try using importlib to import clr from pythonnet with full
path provided to clr.pyd.

**Comment by satanasss** [comment]
Wow thanks for the quick answer ! Indeed getting rid of clr and reinstalling pyhtonnet fixed the stuff.

Many thanks guys !

**Comment by vmuriart** [comment]
In retrospective we should have registered that package name in PyPi

**Comment by denfromufa** [comment]
I think there is a way to "complain" to pypi members, so that this clashing is resolved.

This clr package is only about one month old.

But let's wait over the weekends, if the author responds.

**Comment by filmor** [comment]
I've pinged him again to get to a decision: https://github.com/lmittmann/clr/issues/1

In case he does not want to give up the name we should:
- Add information to our README.md that one must make sure that `pythonnet` installed via `pip`
- Add the same information to our pypi channel and homepage

In case he does:
- We have to grab the `clr` name on pypi and essentially alias it to `pythonnet`, for example by putting a package there that prints a warning and depends on `pythonnet`

**Comment by filmor** [comment]
@denfromufa @vmuriart I have added you as owners of the `clr` package on PyPI. I don't have the time to finish the work there right now, so here's the TODO list :)

- [ ] Put up a new package information page that is basically our usual README but adds a disclaimer that the previously listed package is now called [`style`](https://pypi.python.org/pypi/style/1.0.3) and that the "real" name of the Python.NET package is `pythonnet`
- [ ] Create a small package (i.e. `setup.py`) within our git-repo that only depends on `pythonnet`
- [ ] Remove the existing `clr` package and push our alias-package

**Comment by denfromufa** [comment]
We may as well reserve Python.net package name on pypi.

**Comment by eabase** [comment]
I just got this error and found out it was due to having multiple Python installations, including `pythonnet` package install in each. 

1. If, you're in Windows, do: **`where.exe python`** to find where it's installed. 
2. Next, use the version specific `pip`, like this:
```bash
pip list
pip3.8.exe list
```
(depending on what you found in (1).)

The version I am using, is `conda` so scripts installed with conda and using `import clr`, fails with:
**`ModuleNotFoundError: No module named 'clr'`**
when ran from command line. 

I don't know how to fix this. 

However, running script in it's own home directory (elsewhere) seem to work. Very weird!
